readyapi4j-core/src/main/java/com/smartbear/readyapi4j/teststeps/mockresponse/SoapMockResponseTestStepBuilder.java

```diff
@@ -1,47 +1,47 @@
 package com.smartbear.readyapi4j.teststeps.mockresponse;
 
-import com.smartbear.readyapi.client.model.WsdlMockResponseTestStep;
+import com.smartbear.readyapi.client.model.SOAPMockResponseTestStep;
 import com.smartbear.readyapi4j.teststeps.TestStepBuilder;
 import com.smartbear.readyapi4j.teststeps.TestStepTypes;
 
 import java.net.URL;
 
-public class WsdlMockResponseTestStepBuilder implements TestStepBuilder<WsdlMockResponseTestStep> {
-    private final WsdlMockResponseTestStep mockResponseTestStep = new WsdlMockResponseTestStep();
+public class SoapMockResponseTestStepBuilder implements TestStepBuilder<SOAPMockResponseTestStep> {
+    private final SOAPMockResponseTestStep mockResponseTestStep = new SOAPMockResponseTestStep();
```

Comment:
Sorry, I mixed up the text we use for test step type and the mdoel name. Fixing it. Thanks